
pedsim:
    teleop_flag: 0
    update_rate: 25
    simulation_factor: 1
    max_robot_speed: 2.0
    robot_wait_time: 10

    # Social force model
    obstacle_force: 10.0
    obstacle_sigma: 0.2
    social_force: 5.1
    group_gaze_force: 3.0
    group_coherence_force: 2.0
    group_repulsion_force: 1.0
    random_force: 0.1
    along_wall_force: 2.0

    enable_groups: 1
    group_size_lambda: 1.3
    wait_time_beta: 0.2

    # world parameters
    cell_height: 1
    cell_width: 1
